entrypoint.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
-echo "Using SERVICE_CONFIG_HASH = $SERVICE_CONFIG_HASH"
-echo $(jq --arg SERVICE_CONFIG_HASH "$SERVICE_CONFIG_HASH" '.serviceRegistry.configHashes = [ $SERVICE_CONFIG_HASH ]' snapshot.json) > snapshot.json
-yarn hardhat node
+if [[ -n "$SERVICE_CONFIG_HASH" ]]; then
+  echo "Using SERVICE_CONFIG_HASH = $SERVICE_CONFIG_HASH"
+  echo $(jq --arg SERVICE_CONFIG_HASH "$SERVICE_CONFIG_HASH" '.serviceRegistry.configHashes = [ $SERVICE_CONFIG_HASH ]' snapshot.json) > snapshot.json
+  yarn hardhat node
```

Comment:
Move this out of the `if` block

Reply:
You mean the last line? If we run it with the empty config hash, it will just fail, since the first service registry creation function will check for the hash length and revert.

Reply:
It'll still work since we will still hash in the config hash in `snapshot.json`, I tried this locally and it worked